src/mixins/filing-template-mixin.ts

```diff
@@ -945,8 +945,8 @@ export default class FilingTemplateMixin extends DateMixin {
 
     // store document service state
     this.setDocumentIdState({
-      valid: filing.header.documentIdState.valid,
-      consumerDocumentId: filing.header.documentIdState.consumerDocumentId
+      valid: filing.header.documentIdState?.valid,
```

Comment:
Do you need this to fix the unit tests, but it works fine when deployed?

If so, a better solution might be to update the unit tests so the subject object is declared.

(I am bit wary of optional chaining because it can mask errors. I'm not saying that's the case here but I try to evaluate all uses of it. Let me know what you think.)